binary-compatibility-validator/reference-public-api/ktor-client-core.txt

```diff
@@ -1169,3 +1154,21 @@ public final class io/ktor/client/utils/HeadersKt {
 	public static synthetic fun buildHeaders$default (Lkotlin/jvm/functions/Function1;ILjava/lang/Object;)Lio/ktor/http/Headers;
 }
 
+public final class io/ktor/network/sockets/ConnectTimeoutException : java/net/ConnectException {
+	public fun <init> (Lio/ktor/client/request/HttpRequestData;)V
```

Comment:
If we are going to commonize these exceptions whether it makes sense to pass `HttpRequestData` as a parameter?